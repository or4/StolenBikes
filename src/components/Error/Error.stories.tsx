import * as React from 'react';
import { storiesOf } from '@storybook/react';

import { Error } from './Error';

const stories = storiesOf('Error', module);

stories.add('Error', () => {
    return <Error />;
});
