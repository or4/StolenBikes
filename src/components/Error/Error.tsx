import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
    color: red;
`;

export const Error: React.FC = () => <Container>Ooops, something went wrong</Container>;
