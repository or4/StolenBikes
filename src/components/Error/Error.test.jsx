import React from 'react';
import { mount } from 'enzyme';

import { Error } from '../Error';

describe('Error', () => {
    it('should render correctly', () => {
        const component = <Error />;

        const container = mount(component);
        expect(container).toMatchSnapshot();
        container.unmount();
    });
});
