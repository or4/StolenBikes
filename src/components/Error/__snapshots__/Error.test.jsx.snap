// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render correctly 1`] = `
<Error>
  <styled.div>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "dQCClZ",
            "rules": Array [
              "
    margin-top: 40px;
    color: red;
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <div
        className="sc-bdVaJa dQCClZ"
      >
        Ooops, something went wrong
      </div>
    </StyledComponent>
  </styled.div>
</Error>
`;
